### DISCUSSION TITLE
Wie kann man Kosten pro 100km anzeigen?

### DISCUSSION THREAD

**Discussion by horst789github** [teslamate-org/teslamate#3334] [Getting started]
Hi,

wie kann ich bei den Statistiken die Kosten pro 100 km anzeigen lassen? Also aktuell sehe ich die gesamtkosten, dazu würde ich gern noch 100 km kosten. Geht sowas?

**Comment by parkr** [comment]
Which version of TeslaMate are you using? I see Cost per 100mi in my "Charging Statistics" page:

![image](https://github.com/adriankumpf/teslamate/assets/237985/733f32c4-55c0-47f5-be5f-0b5ac4bb4683)

**Reply by horst789github** [reply]
last version, sorry i mean the "Trip" and "Statistics" page.
